packages/react/src/exchanges/hooks/types/useExchangeBookRequest.ts

```diff
@@ -0,0 +1,10 @@
+import type {
+  Config,
+  PostExchangeBookRequestData,
+} from '@farfetch/blackout-client';
+
+export type UseExchangeBookRequestOptions = {
+  enableAutoFetch?: boolean;
+  fetchConfig?: Config;
+  bookRequestData?: PostExchangeBookRequestData;
```

Comment:
Same logic as in the useExchange hook, the user needs to specify it in the `create` action and not in the hook itself.

Reply:
Done.